assets/diagrams/assets.mmd

```diff
@@ -0,0 +1,9 @@
+graph LR
+A(fa:fa-server warehouse.ai)
+B(fa:fa-opencart web service)
+C{fa:fa-database No-SQL Database}
+D[fa:fa-laptop browser]
```

Comment:
Is there a way to get the text vertically centered in these diagrams? I could see the boxes being a design choice to put the text at the bottom, but the diamonds just look weird that way.

<img width="576" alt="Screen Shot 2019-05-08 at 9 00 51 AM" src="https://user-images.githubusercontent.com/15640215/57389889-f8bdaf00-716f-11e9-8a4d-40294177c007.png">

Reply:
it's just the first diagram, it is missing icons atm. It's just an SVG though, so we can style in however we'd like